data.sql

```diff
@@ -9,4 +9,19 @@ INSERT INTO animals VALUES (4, 'Squirtle' ,'1993-4-2', 3, false,-12.13);
 INSERT INTO animals VALUES (5, 'Angemon' ,'2005-6-12', 1, true,-45);
 INSERT INTO animals VALUES (5, 'Boarmon' ,'2005-6-7', 7, true,20.4);
 INSERT INTO animals VALUES (5, 'Blossom' ,'1998-10-13', 3, true,17);
-INSERT INTO animals VALUES (5, 'Ditto' ,'2022-5-14', 4, true,-22);
\ No newline at end of file
+INSERT INTO animals VALUES (5, 'Ditto' ,'2022-5-14', 4, true,-22);
+
+INSERT INTO owners (full_name, age) VALUES 
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38)
+;
+
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
+
```

Comment:
- [ ] I believe you forgot to add some data to this file. If you check the requirements, you should see that all animals should have an owner assigned, but I cannot find this data in your file.
Just to clarify a bit, this is what it should look like, more or less:
```
UPDATE animals
SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell'
Where name = 'Agumon'
```
This is just a sample query, you can use this or any other you prefer, it should accomplish the same function.